src/editor-service/editor-server.ts

```diff
@@ -21,74 +22,78 @@ import {LocalEditorService} from './local-editor-service';
 import {Request, RequestWrapper, ResponseWrapper, SettledValue} from './remote-editor-protocol';
 
 
+
 /**
- * Runs out of process and handles decoded Requests.
+ * Runs out of process and communicates with EditorProcess.
  */
 class EditorServer {
-  private _localEditorService: LocalEditorService;
-  constructor(basedir: string) {
-    this._localEditorService = new LocalEditorService({
-      urlLoader: new FSUrlLoader(basedir),
-      urlResolver: new PackageUrlResolver()
-    });
-  }
+  private _editorService: LocalEditorService;
 
   async handleMessage(message: Request): Promise<any> {
+    if (message.kind === 'init') {
+      if (this._editorService) {
```

Comment:
We don't, and I agree. I haven't dived into logging at all. Recommend punting this right now.